pkg/filter/tracing/tracing.go

```diff
@@ -29,7 +29,7 @@ import (
 	"github.com/apache/dubbo-go-pixiu/pkg/tracing"
 )
 
-const jaegerTraceIDInHeader = "uber-trace-id"
+const TraceIDInHeader = "uber-trace-id"
```

Comment:
change the value to "dubbo-go-trace-id"?

Reply:
"uber-trace-id" is used by jaeger. Since we can support multiple exporters, "dubbo-go-trace-id" is better actually.

Reply:
so, pls change it.